Backend/server.js

```diff
@@ -61,7 +63,48 @@ app.get('/stock', (req, res) => {
     });
 });
 
+/**
+ * Fetches stock symbols based on the search query
+ */
+app.get('/searchResults/:query', (req, res) => {
+    const searchQuery = req.params.query;
+    
+    const url = `https://www.alphavantage.co/query?function=SYMBOL_SEARCH&keywords=${searchQuery}&apikey=${apiKey}`;
```

Comment:
You could have an Alphavantage object and have functions that make these calls to have all of the third party information held in one object.  Less hardcoding overall.

Reply:
I kind of see where you're getting at if this is in reference to the hardcode I applied to the objects with keys like [2. Name]. 
Originally, I was parsing through the json data from the mock data since that what was going to be coming from the api requests. In hindsight, I could organize all of those parameters into an Alphadvantage Object. I don't think that was possible with the mockdata since each piece of mockdata was coming from separate js files. 

I'll troubleshoot this issue today.

Reply:
Think of code as a leave it a little bit better every time you are in it.  I have stepped into some really difficult codebases, but through documentation as well as fixing bad standards with newer patterns, allows it to grow in a more productive manner.